website/docs/d/agent_config.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "consul"
+page_title: "Consul: consul_agent_config"
+sidebar_current: "docs-consul-data-source-agent-config"
+description: |-
+  Provides the configuration information of the local Consul agent.
+---
+
+# consul_agent_config
```

Comment:
Maybe we want to include a `~> **NOTE:**` regarding how this data source differs from `consul_agent_self` (and add something reflective into `consul_agent_config`)?

Reply:
Good call, added that!